src/swift/Source.swift

```diff
@@ -174,7 +174,7 @@ extension DispatchSource {
 	}
 #endif
 
-#if !os(Linux) && !os(Android) && !os(Windows)
+#if !os(Linux) && !os(Android) && !os(Windows) && !os(FreeBSD)
```

Comment:
...and then we're going to have to eventually `&& !os(OpenBSD)` -- should we just bite the bullet and rewrite this `os(macOS)`? (throughout)